lib/target/sms/classic/sms_crt0.asm

```diff
@@ -220,13 +216,14 @@ gg_palette:
     defw 0x0f0f             ;ff 00 ff
     defw 0x0555             ;55 55 55
     defw 0x0fff             ;ff ff ff
+ENDIF
 
     DEFC SpriteSet          = 0       ; 0 for sprites to use tiles 0-255, 1 for 256+
     DEFC NameTableAddress   = $3800   ; must be a multiple of $800; usually $3800; fills $700 bytes (unstretched)
     DEFC SpriteTableAddress = $3f00   ; must be a multiple of $100; usually $3f00; fills $100 bytes
 
 _Data: 
-    defb @00000100,$80 
+    defb @00000110,$80
```

Comment:
The RetroCopy_Win_0.960 emulator doesn't display the screen correctly unless bit 1 is set. Seems to have no adverse effects on the 3 other emulators I tried.